src/features/ProfilePage/components/AccountInfo.tsx

```diff
@@ -0,0 +1,245 @@
+import styled, { css } from 'styled-components';
```

Comment:
Lot of things going on in this file, but probably will get easier when splitting up in the next PR

Reply:
Yes, I promise it'll get better!